emails/views.py

```diff
@@ -604,6 +609,12 @@ def _handle_bounce(message_json):
     bounce = message_json.get('bounce')
     bounced_recipients = bounce.get('bouncedRecipients')
     for recipient in bounced_recipients:
+        recipient_address = recipient.pop('emailAddress', None)
+        recipient_address = parseaddr(recipient_address)[1]
+        recipient_domain = recipient_address.split('@')[1]
+        capture_message(
+            f'bounced recipient domain: {recipient_domain}, {recipient}'
+        )
         try:
             user = User.objects.get(email=recipient.get('emailAddress'))
```

Comment:
```suggestion
            user = User.objects.get(email=recipient.get(recipient_full_address))
```

Reply:
Should we early return if the `email_full_address` was `None` or is this handled in the exception for `User.DoesNotExist` because we'll never have a `User` without an `email` value?